debian/rules

```diff
@@ -4,4 +4,4 @@
 #export DH_VERBOSE = 1
 
 %:
-	dh $@ --with autoreconf,systemd
```

Comment:
I don't think you want to remove this (and the Build-Depends). dh-systemd helps to manage the runtime state of systemd units. So, even though dh_systemd_enable is no longer being used to do the installation, dh_systemd_enable and dh_systemd_start will be used to enable/disable and start/restart/stop the units from the maintainer scripts.

If you want to keep the services from being enabled by default, I suggest you just add an override section for dh_systemd_enable:
```
override_dh_systemd_enable:
        dh_systemd_enable --no-enable
```
See also https://github.com/endlessm/eos-boot-helper/blob/master/debian/rules.